deps.edn

```diff
@@ -0,0 +1,16 @@
+{:paths ["src/clj" "src/cljc" "src/cljs"]
```

Comment:
Maintaining both deps.edn and project.clj is error prone. WDYT about just moving the whole project to tools.deps (deps.edn)?

Reply:
I like the idea but it would require somebody at Nubank to migrate the lib release automation, right?

Reply:
ah right, then yeah I can try to do that